src/screens/OrganizationPeople/OrganizationPeople.test.tsx

```diff
@@ -941,10 +825,10 @@ describe('Organization People Page', () => {
       </MockedProvider>,
     );
     await wait();
-    userEvent.click(screen.getByLabelText(/Users/i));
-    await wait();
-    expect(screen.getByLabelText(/Users/i)).toBeChecked();
-    await wait();
+    // userEvent.click(screen.getByLabelText(/Users/i));
```

Comment:
@duplixx why have you commented the tests ?

Reply:
these tests are not used as by requirements we dont pagination, radio buttons and type of member label